core/src/main/java/org/mule/VoidMuleEvent.java

```diff
@@ -40,7 +40,12 @@ public static VoidMuleEvent getInstance()
         return instance;
     }
 
-    private VoidMuleEvent()
+    public static boolean isVoid(MuleEvent event)
```

Comment:
Not sure about it. We discussed a similar issue the other day related to NullPayload checks.